NSS-Day-Cohort-66/rare-client-golden-droids-1
# Added CSS to app and PostDetails View

I added Post Details View which will show the details for a single post.

## Steps to Test
1. Pull down `ts-PostDetails` branch and switch to it
2. Be sure to pull down `ts-post-serializer-update` on the server side
3. Make sure your debugger is running
4. Login with the following credentials
```
{
        "username": "phifertristan",
        "password": "phifer"
}
```
5. Click on All Posts in Navbar
6. Then click on a post
7. You should see the following:
    - Post Title
    - Header Image
    - Author name
    - Date created
    - Content
8. You should see a button for View Comments and Add Comment, make sure those buttons take you to the appropriate routes

### Added CSS to application
1. click through the app and see if you're okay with the CSS

closes #5
